upload_movie.py

```diff
@@ -0,0 +1,55 @@
+import numpy as np
+import os
+import pandas as pd
+import re
+import logging
+from imdb import IMDb
+from multiprocessing import Pool
+from pymongo import MongoClient
+
+
+RELEVANT_KEYS = ['imdbID', 'title', 'genres', 'votes', 'plot', 'synopsis']
+
+
+def get_connection_url(user, password, url):
+    return "mongodb://{user}:{password}@{url}/".format(user=user, password=password, url=url)
+
+
+def enrich_movie(movie_id, only_english=True):
```

Comment:
1. Please don't forget type hints
2. **Why don't use load_from_imdb here.** It would make much more sense. You will avoid duplicating code and you will return a Movie object so we can keep track of the data that flows to the database